onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -498,6 +498,28 @@ def _onnx_infer_single_node(self, node):
                     if (name in self.initializers_ and name not in self.graph_inputs_)
                 ]
 
+            if node.op_type in [
+                "Add",
+                "Sub",
+                "Mul",
+                "Div",
+                "MatMul",
+                "MatMulInteger",
+                "MatMulInteger16",
+                "Where",
+                "Sum",
```

Comment:
Sum inputs can be between 1 and 2147483647 inputs:
https://onnx.ai/onnx/operators/onnx__Sum.html
Do we need some special handling of 1 or multiple inputs?